src/Horse.Core.Group.Contract.pas

```diff
@@ -28,10 +28,12 @@ interface
     function Put(APath: string; ACallbacks: array of THorseCallback): IHorseCoreGroup<T>; overload;
     function Put(APath: string; ACallbacks: array of THorseCallback; ACallback: THorseCallback): IHorseCoreGroup<T>; overload;
 
+    {$IF CompilerVersion <> 27.0}
```

Comment:
If it doesn't work on XE6, it certainly doesn't work on previous versions. So the directive should look like this:
`{$ IF CompilerVersion > 27.0}`

Reply:
Adjustment performed